documentation/ame-guide/modules/ROOT/pages/namespace/workspace.adoc

```diff
@@ -8,40 +8,41 @@ By default, the Aspect Model Editor file directory is located at following path:
 * `+C:\Users\<YOUR-USER-NAME>\aspect-model-editor+` (on Windows)
 * `+/Users/<YOUR-USER-NAME>/aspect-model-editor+` (on Mac or Linux)
 
-There you can add the folder structure for all your Aspect Models.
-On your local file directory you can manually create such folders for the different namespaces and the version.
-Of course creating and deleting namespaces is also possible with this editor**.**
+On your local file directory, you can manually add, edit, and delete the folder structure for all your Aspect Models.  
 
-Structure is visible in the xref:getting-started/ui-overview.adoc#workspace-sidebar[Workspace sidebar].
+Of course, this is also possible with the Aspect Model Editor. The same folder structure is displayed in the xref:getting-started/ui-overview.adoc#workspace-sidebar[Workspace sidebar].
 
-image::namespace-file-structure.png[width=80%]
+image::namespace-file-structure.png[width=85%, link=self]
 
-Namespaces that do not follow this file structure might not work and might break the references feature.
+NOTE: Namespaces that do not follow this file structure might not work and might break the xref:namespace/use-references.adoc[references] feature.
 
 == Action dialog in workspace
 
 The currently loaded Aspect Model is highlighted gray.
 
-The ... icon next to a model opens the options to
+The *...* icon next to a model opens the options to
 
-* *Open in new windows* - opens the Aspect Model in a new window
+* *Open in new window* - opens the Aspect Model in a new window
```

Comment:
in a similar list (File > New) we did not use a - 
--> maybe use the same pattern for both